pkgs/build-support/dotnet/build-dotnet-module/hooks/default.nix

```diff
@@ -53,7 +53,7 @@
   dotnetFixupHook = callPackage ({ }:
     makeSetupHook {
       name = "dotnet-fixup-hook";
-      deps = [ dotnet-runtime makeWrapper ];
```

Comment:
Is makeWrapper unused in .net packages?

Reply:
It isn't, but `makeSetupHook`'s `deps` attribute maps to `propagatedBuildInputs`: https://github.com/NixOS/nixpkgs/blob/2c7a74c9925187476c7cd4f5dd227742b0c9a19a/pkgs/build-support/trivial-builders.nix#L537

That causes the following error:
```console
~/nix/nixpkgs > nix build -Lf . pkgsCross.aarch64-multiplatform.depotdownloader
error: attribute 'runtimeShell' missing

       at /home/ivv/nix/nixpkgs/pkgs/top-level/all-packages.nix:784:17:

          783|       substitutions = {
          784|         shell = targetPackages.runtimeShell;
             |                 ^
          785|         passthru.tests = tests.makeWrapper;
```
`makeWrapper` is also already provided through `buildDotnetModule`'s `nativeBuildInputs`: https://github.com/NixOS/nixpkgs/blob/2c7a74c9925187476c7cd4f5dd227742b0c9a19a/pkgs/build-support/dotnet/build-dotnet-module/default.nix#L99